eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -27,8 +27,8 @@
 {% block ezxmltext_field %}
 {% spaceless %}
     {% set field_value = field.value.xml|xmltext_to_html5 %}
-    {{ block( 'simple_block_field' ) }}
 {% endspaceless %}
+{{ block( 'simple_block_field' ) }}
```

Comment:
> the purpose I assume to be for cleanliness of the generated HTML.

Yes, but correct behavior is always better than *clean* HTML. So here you can just remove the tag :wink: . 

If you really want to nitpick, you can also remove WS generated by the `set` tag:

```jinja
{% block ezxmltext_field %}
    {%- set field_value = field.value.xml|xmltext_to_html5 -%}
    {{ block( 'simple_block_field' ) }}
{% endblock %}
```

I guess the same is applicable for richtext.